openshift-ci/deploy_metallb.sh

```diff
@@ -3,8 +3,19 @@
 metallb_dir="$(dirname $(readlink -f $0))"
 source ${metallb_dir}/common.sh
 
+# Extract and format the oc version to branch
+extract_version() {
```

Comment:
what's the format of the oc version command's output?

Reply:
That is:
```
Client Version: 4.11.0
Kustomize Version: v4.5.4
Server Version: 4.11.0-0.nightly-2023-06-28-183440
Kubernetes Version: v1.24.15+990d55b
```